dll/win32/msi/package.c

```diff
@@ -1330,6 +1330,58 @@ UINT msi_set_original_database_property( MSIDATABASE *db, const WCHAR *package )
     return r;
 }
 
+#ifdef __REACTOS__
+BOOL WINAPI ApphelpCheckRunAppEx(HANDLE FileHandle, PVOID Unk1, PVOID Unk2, PWCHAR ApplicationName, PVOID Environment, USHORT ExeType, PULONG Reason, PVOID *SdbQueryAppCompatData, PULONG SdbQueryAppCompatDataSize,
```

Comment:
ok that superlong line is cringe. I know MSI comes from wine and does that, but this is a ROS-specific code so you can put each of the arguments on their own line. + Same comment for the next prototype.

Reply:
fits on my screen.
Having a 10+ line block for one prototype is cringe.